tools/packaging/kata-deploy/helm-chart/README.md

```diff
@@ -0,0 +1,117 @@
+# Kata Containers Deploy – Helm Chart
+
+A Helm chart that installs the kata‑deploy DaemonSet and its helper assets,
+enabling Kata Containers runtimes on your Kubernetes, K3s, RKE2, or K0s cluster.
+
+## TL;DR
+
+```sh
+# Install directly from the official release tarball
+# update the release X.YY.Z to your needs or just use the latest
+
+export VERSION=$(curl -sSL https://api.github.com/repos/kata-containers/kata-containers/releases/latest | jq .tag_name | tr -d '"')
+
+helm install kata-deploy \
+  https://github.com/kata-containers/kata-containers/releases/download/${VERSION}/kata-deploy-${VERSION}.tgz
+
+# See everything you can configure
+helm show values \
+  https://github.com/kata-containers/kata-containers/releases/download/${VERSION}/kata-deploy-${VERSION}.tgz
+```
+
+## Prerequisites
+
+- **Kubernetes ≥ v1.22** – v1.22 is the first release where the CRI v1 API
+  became the default and RuntimeClass left alpha.  The chart depends on those
+  stable interfaces; earlier clusters need feature‑gates or CRI shims that are
+  out of scope.
+
+- CRI‑compatible runtime (containerd or CRI‑O)
+
+- Nodes must allow loading kernel modules and installing Kata artifacts (the
+  chart runs privileged containers to do so)
+
+## Installing Helm
+
+If Helm is not yet on your workstation or CI runner, install Helm v3 (v3.9 or
+newer recommended):
+
+```sh
+# Quick one‑liner (Linux/macOS)
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
+# Or via your package manager
+sudo apt-get install helm        # Debian/Ubuntu
+brew install helm                # Homebrew on macOS / Linuxbrew
+```
+
+Verify the installation:
+
+```sh
+helm version
+```
+
+## Installing the Chart
+
+```sh
+
+helm install kata-deploy \                           # release name
```

Comment:
Is it worth suggesting doing `show values` and/or linking to the configuration reference below prompting users to pick the correct k8sDistribution/shim(s)/other config in case the user is like me and run the commands without reading the whole doc!

Reply:
RTFM! :) Done added a comment and like to lookup the configuration reference.